extensions/applicationinsights-react-native/Tests/Unit/src/reactnativeplugin.tests.ts

```diff
@@ -99,8 +97,8 @@ export class ReactNativePluginTests extends TestClass {
         this.testCase({
             name: 'Autocollection is enabled by default',
             test: () => {
-                const autoCollectStub = this.sandbox.stub(this.plugin, '_collectDeviceInfo');
-                const autoCollectExceptionStub = this.sandbox.stub(this.plugin, '_setExceptionHandler', () => true);
+                const autoCollectStub = this.sandbox.stub(this.plugin as any, '_collectDeviceInfo');
+                const autoCollectExceptionStub = this.sandbox.stub(this.plugin as any, '_setExceptionHandler').callsFake(() => true);
```

Comment:
Change required as part of bumping to the latest version of Sinon which removed the previous stub function